### PR TITLE
Build standalone bundle for each submodule

### PR DESCRIPTION
**Pull request by Pessimistress**
For #2736 

#### Change List
- Build standalone bundle for each submodule

### CODE REVIEW

**Comment by Pessimistress** [comment]
> Isn't it better to build/include the script for each submodule in that submodule

That's fair. So

```html
<script src="https://unpkg.com/deck.gl@~7.0.0/core.min.js" />
<script src="https://unpkg.com/deck.gl@~7.0.0/layers.min.js" />
```

will become

```html
<script src="https://unpkg.com/@deck.gl/core@~7.0.0/dist.min.js" />
<script src="https://unpkg.com/@deck.gl/layers@~7.0.0/dist.min.js" />
```

File: modules/geo-layers/bundle/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const deckGLLayers = require('../src');
```
Comment (ibgreen) [inline modules/geo-layers/bundle/index.js]: Nit: I guess these could just be `src/bundle.js`, save us a directory for just one file?

File: modules/main/bundle/index.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const deck = require('../../core/bundle');
+
+Object.assign(
+  deck,
+  require('@deck.gl/layers'),
+  require('@deck.gl/aggregation-layers'),
+  require('@deck.gl/geo-layers'),
+  require('@deck.gl/mesh-layers'),
+  require('@deck.gl/mapbox'),
+  require('@deck.gl/json')
```
Comment (ibgreen) [inline modules/main/bundle/index.js]: Yay!

File: scripts/bundle.config.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+/**
+ * peerDependencies are excluded using `externals`
+ * e.g. @deck.gl/core is not bundled with @deck.gl/geo-layers
+ */
+
+let externals = {};
+const {peerDependencies = {}} = PACKAGE_INFO;
+
+for (const depName in peerDependencies) {
```
Comment (ibgreen) [inline scripts/bundle.config.js]: This automation looks very nice, though maybe create some functions with self documenting names, and add some one line comments to explain how the automation here works. Maintainers may not be as well versed with everything involved in this as you are right now.

File: modules/geo-layers/bundle/index.js (hunk 1 above)
Comment (Pessimistress) [reply modules/geo-layers/bundle/index.js]: The `core` module bundle contains multiple files.
Files under `src/` are transpiled and distributed, which is a little weird.

**Comment by jingsam** [comment]
Missing 'alias.js' file

**Pull request merged**